templates/jicofo/logging.properties.j2

```diff
@@ -1,4 +1,39 @@
 handlers= java.util.logging.ConsoleHandler
-.level= {{ jitsi_meet_jicofo_loglevel }}
+#handlers= java.util.logging.ConsoleHandler, com.agafua.syslog.SyslogHandler
```

Comment:
I would add the whole content of this file into a variable like `jitsi_meet_jicofo_logging_properties` to ensure we can adjust them in multiple ways. It seems we can't make every logging handler from the application configurable with a variable here in that role. So I would prefer being flexible and allow to override them as a complete file with this change as default.

Reply:
We could do this, but do you really think it's necessary?
What else do we want apart from lowering loglevel and maybe enable syslog?

Reply:
For some reasons you will only enable more verbose logging for a component. Also had experienced that some logging handlers seems to not inherit from the global log level and for this cases it needs to adjust handler by handler.